content/en/observability_pipelines/setup/splunk.md

```diff
@@ -382,6 +391,35 @@ EOT
 }
 ```
 {{% /tab %}}
+{{% tab "CloudFormation" %}}
+To install the Worker in your AWS Account, use the CloudFormation template to create a Stack:
+
+  1. Download the [CloudFormation template][1] for the Worker.
+
+  2. In the **CloudFormation console**, click **Create stack**, and select the **With new resources (standard)** option.
+
+  3. Make sure that the **Template is ready** option is selected. Click **Choose file** and add the CloudFormation template file you downloaded earlier. Click **Next**.
+
+  4. Enter a name for the stack in **Specify stack details**.
+
+  5. Fill in the parameters for the CloudFormation template. A few require special attention:
+
+  * For **APIKey** and **PipelineID**, provide the key and ID that you gathered earlier in the Prerequisites section.
+
+  * For the **SplunkToken**, provide the token you created earlier on your Splunk index.
+    
+  * For the **VPCID** and **SubnetIDs**, provide the subnets and VPC you chose earlier.
+
+  * All other parameters are set to reasonable defaults for a Worker deployment but you can adjust them for your use case as needed.
```

Comment:
```suggestion
      * For **APIKey** and **PipelineID**, provide the key and ID that you gathered earlier in the Prerequisites section.

      * For the **SplunkToken**, provide the token you created earlier on your Splunk index.
    
      * For the **VPCID** and **SubnetIDs**, provide the subnets and VPC you chose earlier.

      * All other parameters are set to reasonable defaults for a Worker deployment but you can adjust them for your use case as needed.
```

Reply:
Same as above.

Reply:
could you update the parameters such that it's formatted code quote instead of bold?